notebooks/emm-20230226-00-onnx2pytorch.ipynb

```diff
@@ -0,0 +1,524 @@
+{
```

Comment:
<p>The other thing to try is to run the onnx conversion again, but this time with opset 17 and a single output corresponding to the embedding layer (242). You'll need to define a onnx_model_v2 here. </p><p> </p><pre class="ql-syntax" spellcheck="false">! python -m tf2onnx.convert \
  --opset 17 \
  --input {opt_graph_v1} \
  --inputs inputs:0 \
  --outputs unknown_242:0 \
  --output {onnx_model_v2}

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>